dogpile/cache/backends/redis.py

```diff
@@ -101,6 +101,13 @@ class RedisBackend(CacheBackend):
     """
 
     def __init__(self, arguments):
+        super().__init__(
+            {
+                "pickler": pickle.dumps,
+                "unpickler": pickle.loads,
```

Comment:
yeah this syntax is messing with me, because it registers as exactly like the syntax that would fail:

   Foo(pickler=pickle.dumps, unpickler=pickle.loads, **arguments)

again my problem not yours...